server/UserRegistration.java

```diff
@@ -0,0 +1,357 @@
+package newbank.server;
+
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.text.DateFormat;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.Scanner;
+
+public class UserRegistration {
+
+    private static Scanner sc = new Scanner(System.in);
+        UserRegistration ur = UserRegistration.newUser();
+    public UserRegistration() throws SQLException {
+    }
+
+    public static UserRegistration newUser() throws SQLException {
+        // set all methods to return values - to then push to db.
+        String UserFullname = UserRegistration.getNameFromUser();
```

Comment:
After removing the `static` keyword from all of the methods, you can then simply do this:
```java
String UserFullname = getNameFromUser();
```